### PR TITLE
Changelog: adds missing enterprise entries

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

A lot of enterprise entries are missing recently in the changelog. Added all I could find.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

We really need to update the tooling to make sure that we get the enterprise changelogs in here as well.

@torkelo, how would we go about doing that the best way? If I'm not mistaken the changelog update is executed as a github action.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1975,7 +2005,7 @@ This option to group query variable values into groups by tags has been an exper
 
 # 7.0.1 (2020-05-26)
 
-### Features / Enhancements
+### Features / Enhancementsrecord
```
Comment (sakjur) [inline CHANGELOG.md:2008]:
```suggestion
### Features / Enhancements
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -481,6 +490,10 @@ Issue [#33879](https://github.com/grafana/grafana/issues/33879)
 - **Dashboard:** Remove support for loading and deleting dashboard by slug. [#35104](https://github.com/grafana/grafana/pull/35104), [@dsotirakis](https://github.com/dsotirakis)
 - **InfluxDB:** Deprecate direct browser access in data source. [#35105](https://github.com/grafana/grafana/pull/35105), [@gabor](https://github.com/gabor)
 - **VizLegend:** Add a read-only property. [#35096](https://github.com/grafana/grafana/pull/35096), [@dprokop](https://github.com/dprokop)
+- **Runtime reload**: Introduce runtime reload for SAML settings (Enterprise)
+- **FGAC**: Introduce fine grained access control (Enterprise)
```
Comment (xlson) [inline CHANGELOG.md:494]: @sakjur what should this say, I'm thinking you might not want it to say FGAC :)

Comment (sakjur) [reply CHANGELOG.md:494]:
```suggestion
- **Access control**: Introduce fine grained access control beta (Enterprise)
```

**Comment by xlson** [comment]
@torkelo So the changelog is re-generated completely each time we do a release? Then we definitely need to update the action to take enterprise into account.

**Comment by torkelo** [comment]
the changelog action will also check enterprise repo and add the `(Enterprise)` suffix to items

**Comment by torkelo** [comment]
@xlson not completely but each release is is rebuilt every time (and both changelog and release note for docs is generated)

**Comment by xlson** [comment]
Okay, I've change our label from `add-to-changelog` to `add to changelog`.

**Comment by xlson** [comment]
@torkelo Thanks, it works now that we've fixed the label.

**Comment by torkelo** [comment]
great

**Comment by xlson** [comment]
Closing this in favor of updating the changelog the correct way.

**Comment by marefr** [comment]
Good to know :+1:

**Pull request closed without merging**